test/codeu/chat/server/DatabaseTest.java

```diff
@@ -5,15 +5,17 @@
 import database.Connector;
 import java.util.UUID;
 import org.junit.Test;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
 
 
 public final class DatabaseTest {
 
-  private Connector con = new Connector();
 
   @Test
   public void testAddAndDeleteUser() {
-    System.out.println("DATABASE TEST RUN");
+    Connector con = new Connector();
```

Comment:
There's an `@Before` annotation that I believe can be invoked on the `setup()` method — this will run before each test. There's also a `@After` for cleanup / teardown operations.